src/main/java/gov/usgs/wma/waterdata/groundwater/RdbWriter.java

```diff
@@ -77,10 +77,25 @@ public RdbWriter writeRow(DiscreteGroundWater dgw) {
 		}
 		writeValue(  4, time);
 
+		// This is included for the benefit of the code reviewers and will be removed
+		// before checkin.  It is the old implementation, and is replaced by the four
+		// lines after the commented out section.
+		/*if (StringUtils.isEmpty(dgw.levelFeetBelowLandSurface)) {
+			writeValue(  7, "");
+			writeValue(  1, "S"); // entry code for above Sea
+			writeValue( 10, dgw.verticalDatumCode);
+			writeValue(  8, dgw.levelFeetAboveVerticalDatum);
+		} else {
+			writeValue(  7, dgw.levelFeetBelowLandSurface);
+			writeValue(  1, "L"); // entry code for below Land
+			writeValue( 10, "");
+			writeValue(  8, "");
+		}*/
 		writeValue(  7, "");
-		writeValue(  1, "");
+		writeValue(  1, "S");
 		writeValue( 10, dgw.verticalDatumCode);
-		writeValue(  8, "");
+		writeValue(  8, dgw.displayResult);
```

Comment:
Is displayResult similar to levelFeetAboveVerticalDatum or levelFeetBelowLandSurface ?
Maybe we should just have placed that value in the filed that it represents and let this go unchanged?
no sure.

Reply:
yes, we need to map display result to `levelFeetAboveVerticalDatum` and `levelFeetBelowLandSurface`

Reply:
@kkehl-usgs we can work together on this if you like.

Reply:
Updated with changes based on pair session.